### DISCUSSION TITLE
reset_at does not update correctly

### DISCUSSION THREAD

**Discussion by frlefebvre** [trakt/trakt-api#412] [Q&A] [answered]
Hi there,

I've been trying to account for rewatches but the reset_at value seems to lag behind when querying the Watched endpoint. Here is what I've observed:
1. Reset a show on the Trakt website: reset_at is still null.
2. Add additional replay for an episode from that show: reset_at is now showing the right date.
3. Undo the rewatch from Trakt: reset_at still shows its previous date.
4. Add additional replay for an episode from that show: reset_at is now null as expected.

Note that this is happening in staging as I do not have a VIP account in production.

**Comment by rectifyer** [comment, marked as answer]
I will have a fix out for this tomorrow and it will correctly re-cache the `*/watched/shows` after rewatching.

**Reply by ZelKami** [reply]
I think the issue has resurfaced some time ago.
